package/kedro_viz/models/experiment_tracking.py

```diff
@@ -78,11 +90,18 @@ def load_tracking_data(self, run_id: str):
             # tracking datasets do not have load methods defined yet but would be the
             # same as json loader.
             # pylint: disable=import-outside-toplevel
-            from kedro.extras.datasets import json, tracking
+            from kedro.extras.datasets import json, matplotlib, plotly, tracking
 
             tracking.JSONDataSet._load = json.JSONDataSet._load  # type: ignore
             tracking.MetricsDataSet._load = json.JSONDataSet._load  # type: ignore
-            self.runs[run_id] = self.dataset.load()
+            plotly.JSONDataSet._load = json.JSONDataSet._load  # type: ignore
+            plotly.PlotlyDataSet._load = json.JSONDataSet._load  # type: ignore
+            matplotlib.MatplotlibWriter._load = matplotlib_writer_load  # type: ignore
```

Comment:
Just thought of a better way to do this. Move this to the top of the file where imports normally go like this:
```
with suppress(ImportError):
    # tracking datasets do not have load methods defined yet but would be the
            # same as json loader.
            # pylint: disable=import-outside-toplevel
            from kedro.extras.datasets import json, matplotlib, plotly, tracking

with suppress(NameError):
            tracking.JSONDataSet._load = json.JSONDataSet._load  # type: ignore
            tracking.MetricsDataSet._load = json.JSONDataSet._load  # type: ignore
            plotly.JSONDataSet._load = json.JSONDataSet._load  # type: ignore
            plotly.PlotlyDataSet._load = json.JSONDataSet._load  # type: ignore
            matplotlib.MatplotlibWriter._load = matplotlib_writer_load  # type: ignore
```

Reply:
this doesn't work for matplotlib_writer_load. so i will keep it as is and we can refactor it later.

Reply:
Played around with this and got it working in [04165de](https://github.com/kedro-org/kedro-viz/pull/984/commits/04165de16f0857c066315ad5d5c76824f0ba96d0).